app/src/main/java/ch/sdp/vibester/activity/GameSetupActivity.kt

```diff
@@ -30,67 +29,85 @@ import retrofit2.Response
 class GameSetupActivity : AppCompatActivity(), AdapterView.OnItemSelectedListener {
     var difficulty = "Easy"
     var game = "local_buzzer"
-     lateinit var gameManager: GameManager;
+    var gameSize = "One"
+    lateinit var gameManager: GameManager;
```

Comment:
```suggestion
    lateinit var gameManager: GameManager
```
why a ";" ? we never used them in whole code